lib/Algo2023Day04.cs

```diff
@@ -8,32 +10,112 @@ namespace Lib
 {
     internal class Algo2023Day04 : IAlgo
     {
-        public string Solve(string[] input, bool isBonus = false)
+        public string Solve(string[] input, bool isBonus = true)
         {
-            int totalPoint = 0;
-            foreach (var line in input) 
+            if(isBonus)
```

Comment:
```suggestion
return isBonus ? bonus(input) : standard(input);
```


ou alors, si c'est le seul code de la méthode, on peut aussi avoir:
```suggestion
=> isBonus ? bonus(input) : standard(input);
```